src/ex1.py

```diff
@@ -117,7 +118,7 @@ def map_reads(query: str, reference: str) -> Dict[str, Dict[str, List[int]]]:
 
 
 if __name__ == '__main__':
-    query = '../data/sequences.fasta'
```

Comment:
This only works if you execute the code from a specific location. It's best to always write code such that it can be executed from anywhere (though I didn't insist on this for the code in `run_me.sh` - it's possible in Bash, too, to use the current script file as an anchor, but it's a little complicated)